components/list/list-item-menu-item-mixin.js

```diff
@@ -0,0 +1,154 @@
+import '../colors/colors.js';
+import { css, html } from 'lit';
+import { getUniqueId } from '../../helpers/uniqueId.js';
+import { ListItemMixin } from './list-item-mixin.js';
+import { EventSubscriberController } from '../../controllers/subscriber/subscriberControllers.js';
+
+const interactiveElements = {
```

Comment:
Maybe we should look at factoring these core interactive elements/roles out somewhere for re-use. I know `d2l-tooltip` defines a few extra, but it could combine these core ones with its extra ones. Not sure if they are repeated anywhere else.